### DISCUSSION TITLE
怎么把模型转成ollama的模型,how to convert model to ollama

### DISCUSSION THREAD

**Discussion by wangchao1980** [ymcui/Chinese-LLaMA-Alpaca-2#363] [General]
怎么把模型转成ollama的模型,how to convert model to ollama

**Comment by ymcui** [comment]
我们的模型就是llama-2结构。所以你应该去ollama那边查找如何支持llama-2。

**Comment by Czhazha** [comment]
> 怎么把模型转成ollama的模型,how to convert model to ollama

请问下你有跑通这个在ollama使用的模型吗？我按照ollama的转换，能转成功，但是运行后回复是混乱的。
这个就不知道怎么定位问题了

**Comment by wangchao1980** [comment]
加qq聊吧，23326199，您说的混乱是？&nbsp;


王超
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年1月30日(星期二) 中午1:11
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [ymcui/Chinese-LLaMA-Alpaca-2] 怎么把模型转成ollama的模型,how to convert model to ollama (Discussion #363)





  
怎么把模型转成ollama的模型,how to convert model to ollama
  
请问下你有跑通这个在ollama使用的模型吗？我按照ollama的转换，能转成功，但是运行后回复是混乱的。
 这个就不知道怎么定位问题了
